hat/assets/js/cypress/integration/11 - submissions/list.spec.js

```diff
@@ -156,6 +160,36 @@ describe('Submissions', () => {
             const errorCode = cy.get('#error-code');
             errorCode.should('contain', '401');
         });
+        describe('Search field', () => {
+            beforeEach(() => {
+                goToPage();
+            });
+
+            it('should enable search button', () => {
+                cy.get('[data-test="search-button"]')
+                    .as('search-button')
+                    .should('be.disabled');
+                cy.get('#search-search').type(search);
+                cy.get('[data-test="search-button"]')
+                    .invoke('attr', 'disabled')
+                    .should('equal', undefined);
+            });
+
+            it('should disable search button if search contains forbidden characters', () => {
+                cy.get('[data-test="search-button"]')
+                    .as('search-button')
+                    .should('be.disabled');
+                cy.get('#search-search').type(searchWithForbiddenChars);
+                if (
+                    containsForbiddenCharacter(
+                        searchWithForbiddenChars,
+                        forbiddenCharacters,
+                    )
+                ) {
+                    cy.get('@search-button').should('be.disabled');
+                }
+            });
+        });
```

Comment:
These whole describe can go to a method we can call on each page, instead of copy paste it no ?